src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -120,6 +120,8 @@ public SwitchParameter NoClobber
 
```

Comment:
@IISResetMe, your last commit had 2 failures in `PowerShell-CI-linux`
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got @(@{InputObject=ConvertFrom-Clixml; SideIndicator==>}, @{InputObject=ConvertTo-Clixml; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 542
542:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Write-Verbose; ConfirmImpact=None; SideIndicator==>}, @{Name=Write-Warning; ConfirmImpact=None; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 566
566:         $result | Should -BeNullOrEmpty
```